ui/css/itcss/components/send.scss

```diff
@@ -1,927 +0,0 @@
-@use "design-system";
-
-.send-screen-wrapper {
-  display: flex;
-  flex-flow: column nowrap;
-  z-index: 25;
-
-  @include design-system.screen-sm-max {
-    width: 100%;
-    overflow-y: auto;
-  }
-
-  section {
-    flex: 0 0 auto;
-  }
-}
-
-.send-screen-card {
-  background-color: var(--color-background-default);
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-  padding: 46px 40.5px 26px;
-  position: relative;
-  align-items: center;
-  display: flex;
-  flex-flow: column nowrap;
-  width: 498px;
-  flex: 1 0 auto;
-
-  @include design-system.screen-sm-max {
-    top: 0;
-    width: 100%;
-    box-shadow: none;
-    padding: 12px;
-  }
-}
-
-/* Send Screen */
-
-.send-screen section {
-  margin: 4px 16px;
-}
-
-.send-screen input {
-  @include design-system.H7;
-
-  width: 100%;
-}
-
-.send-eth-icon {
-  border-radius: 50%;
-  width: 70px;
-  height: 70px;
-  border: 1px solid var(--color-border-muted);
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-  position: absolute;
-  top: -35px;
-  z-index: 25;
-  padding: 4px;
-  background-color: var(--color-background-default);
-
-  @include design-system.screen-sm-max {
-    position: relative;
-    top: 0;
-  }
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 57 36 AM](https://github.com/user-attachments/assets/411d24b9-f94b-4960-a356-e0a5197c45eb)